### DISCUSSION TITLE
undefined key `load_ckpt` is not allowed in strict mode

### DISCUSSION THREAD

**Discussion by WenyeXuan** [deepmodeling/dpgen#536] [Q&A]
Hello everyone

I am using dpgen v0.10.1 with deepmd-kit 2.0 gpu version installed by conda on a slurm HPC system. I keep receiving the error "undefined key `load_ckpt` is not allowed in strict mode"

Is it a problem of the input parameter version difference in deepmdkit and dpgen? Because I did not have the load_ckpt comment in my para.json file for dpgen input but it keep appearring in the input.json when running deepmdkit on remote HPC.

Best regards,
Wenye

**Comment by amcadmus** [comment]
plz remove the `load_ckpt` key from the template of training script. 

see also https://github.com/deepmodeling/deepmd-kit/issues/919#issuecomment-894901978
